### DISCUSSION TITLE
Add a mouse lock feature like for Turbowarp

### DISCUSSION THREAD

**Discussion by hehe51** [ScratchAddons/ScratchAddons#5678] [Ideas] [answered]
For 3D games like minecraft or fps games, it allows endless turning instead of limiting the mouse to the stage

**Comment by BroJac5246** [comment, marked as answer]
This won't be added, since it'll cause projects to break for non-SA users.

**Reply by DNin01** [reply]
Yeah, this will allow people to create projects that can only be run (as intended) with the help of Scratch Addons, which isn't ideal. But TurboWarp, a Scratch mod, does have [an extension gallery](https://extensions.turbowarp.org/), and there's a "Pointerlock" extension with this feature, so if you don't mind sharing your projects outside of Scratch, you can use that.
